cypress/e2e/tests/pages/explorer/dashboard/cluster-dashboard.spec.ts

```diff
@@ -208,6 +208,35 @@ describe('Cluster Dashboard', { testIsolation: 'off', tags: ['@explorer', '@admi
   const nsName = `namespace${ +new Date() }`;
 
   it('can view events', () => {
```

Comment:
I think we should mark vai related tests with `@vai` tag. I've been doing that so that we can quickly identify/filter those tests

Reply:
I've converted it to draft, I am still investigating a couple edge caases